lib/defaults/defaults.go

```diff
@@ -1190,6 +1188,9 @@ var (
 	// A failed precondition usually means a configuration error when an operation cannot be retried.
 	// The exit code is used to prevent the agent service from restarting after shutdown
 	FailedPreconditionExitCode = 252
+
+	// MaxExpandConcurrency is the number of servers that can be joining the cluster concurrently
+	MaxExpandConcurrency = (runtime.NumCPU() / 3) + 1
```

Comment:
You recently [tweaked](https://github.com/gravitational/gravity/pull/2222/files#diff-cc34e8b8bef47626c4518b46eb5a56458acedd961d290d5a79b5fe5ff63fa832R1310) this to be `(runtime.NumCPU() / 3) + 4` - is this relevant for 6.1 branch?